project_templates/roundtable_aiohttp_bot/example/requirements/main.in

```diff
@@ -6,8 +6,6 @@
 #     make update-deps
 
 safir
-aiohttp
-aiodns
-cchardet
+aiohttp[speedups]
```

Comment:
So I specifically _didn't_ do this because I had a really hard time building the Brotli package in our container. Hence I list just the useful parts of the speedups extra, namely aiodns and cchardet. Did you manage to get Brotli to build in Docker?

Reply:
Ah! Okay, I was wondering about that. I got it to build outside a container but didn't try in Docker. I'll drop this patch.